src/cli/cli.cpp

```diff
@@ -3155,6 +3155,25 @@ void Interpreter::HandlePingReply(const otPingSenderReply *aReply)
     OutputLine(": icmp_seq=%d hlim=%d time=%dms", aReply->mSequenceNumber, aReply->mHopLimit, aReply->mRoundTripTime);
 }
 
+void Interpreter::HandlePingStatistics(const otPingSenderStatistics *aStatistics, void *aContext)
+{
+    static_cast<Interpreter *>(aContext)->HandlePingStatistics(aStatistics);
+}
+
+void Interpreter::HandlePingStatistics(const otPingSenderStatistics *aStatistics)
+{
+    double packetLossRate = 1 - static_cast<double>(aStatistics->mReceivedCount) / aStatistics->mTotalPingCount;
```

Comment:
Noticed that CLI builds (ftd, mtd) has about 2.4KB code size increase from this change. My guess/theory is that it may be due to performing `double` division here to calculate the loss rate.

My suggestion is to use the int (`uint32_t`) division instead and avoid using `double` calculation.
   - We can multiply the `mReceivedCount` by `1000` and then divide using `uint32_t` division.
    - Result `/ 10` gives the percentage and the result `% 10` give the one digit after decimal point `.`